### DISCUSSION TITLE
What's the correct way to run flexget after adding a new show?

### DISCUSSION THREAD

**Discussion by kerryandjane** [Flexget/Flexget#3484] [Q&A]
Hi, i've been writing a lot of scripts recently trying out a few things. During test phases I needed flexget to produce outputs regularly. 

I've been trying to add new series to my config for Flexget to download but can't get flexget to re-parse the rss entries.
i've had to go the `flexget database reset --sure` route but I lose webserver log in functionality everytime I do. (not the point)

What's the correct way to add a series to the list and parse rss entries, looking for the newly added series?

Can't find it anywhere on the plugins page or any of the CLI help pages. I'm sure it's easy but can't figure it out myself.

Here's what i'm doing so far.

1. Adding a series to my series list in config.yml, saving it.
2. Executing the task (all variations of options)
3. Seeing that a new entry has been added to `flexget series list`

Entries are never rerun. No luck.

Trying to avoid having to "inject".

Thanks in advance.

**Comment by lamski** [comment]
`flexget daemon reload` is the command I run after changing anything in my `config.yml`.
